protocol/contracts/beanstalk/metadata/MetadataFacet.sol

```diff
@@ -27,26 +33,35 @@ contract MetadataFacet is MetadataImage {
      * Deposits are stored as a mapping of a uint256 to a Deposit struct.
      * ERC20 deposits are represented by the concatination of the token address and the stem. (20 + 12 bytes).
      */
-    function uri(uint256 depositId) external view returns (string memory) {
+    function uri(uint256 depositId) public view returns (string memory) {
```

Comment:
Why was this make `public`? It doesn't seem to be used anywhere else.

Reply:
fixed: 4e04afe74843ddd502168d255a4fd326e49b41ec